frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -149,12 +158,24 @@ export const EmbedFrame = ({
     : [];
   const hasVisibleParameters = visibleParameters.length > 0;
 
-  const hasHeader = Boolean(finalName || dashboardTabs);
+  const hasHeader = Boolean(finalName || dashboardTabs) || downloadsEnabled;
   const isParameterPanelSticky =
     !!dashboard &&
     theme !== "transparent" && // https://github.com/metabase/metabase/pull/38766#discussion_r1491549200
     isParametersWidgetContainersSticky(visibleParameters.length);
 
+  // TODO: pass this as headerActions  from PublicDashboard ?
```

Comment:
```suggestion
```
I'm not sure this is really needed, I'd remove the comment if you're ok

Reply:
I don't think we really discussed it, I think it was just a comment I left there because I wanted to do that, but I'm not sure anymore